docs/data-tunnel/02-tunnel-architecture.md

```diff
@@ -1 +1,47 @@
 # Tunnel Architecture
+
+## Overview
+
+The **Data Tunnel** serves as a bridge for relaying price data from a concurrent price stream to decentralized applications. It ensures seamless interoperability between various blockchain networks, including Cosmos SDK applications, L1/L2+ EVM contracts, and non-EVM contracts.
+
+![Band Tunnel Architecture](../../static/img/band_tunnel_architecture.png)
+
+## Components
+
+### 1. **Concurrent Price Stream**
+
+The Concurrent Price Stream is responsible for obtaining real-time pricing data from validators. The stream continuously aggregates price data from multiple trusted validators, ensuring that the Data Tunnel always has access to high-quality and reliable price feeds. The **Data Tunnel** actively collects this price information from the Concurrent Price Stream for processing and transmission. Through this mechanism, BandChain maintains an up-to-date price oracle that enables Data Tunnels to efficiently relay price data to various blockchain networks.
```

Comment:
`The **Data Tunnel** actively collects this price information from the Concurrent Price Stream for processing and transmission. Through this mechanism, BandChain maintains an up-to-date price oracle that enables Data Tunnels to efficiently relay price data to various blockchain networks.`

this section should be in `Data Tunnel` section instead?